index.bs

```diff
@@ -1507,6 +1508,83 @@ these steps:
 
 </div>
 
+## Drag and Drop ## {#drag-and-drop}
+
+<xmp class=idl>
+partial interface DataTransferItem {
+    FileSystemHandle? getAsFileSystemHandle();
```

Comment:
thinking more about it, and considering the implementation difficulties we're running into to make it possible for this method to be synchronous, we should probably change this to return a `Promise<FileSystemHandle?>`. That way implementations have more freedom to figure out when to do certain checks. (you'll also need to update the domintro and example to work with returned promises).